### ISSUE TITLE
CMake-built mm_pbsa.pl dies during unit tests

### ISSUE THREAD

**Issue by multiplemonomials** [Amber-MD/cmake-buildscripts#19]
I have no idea why this is happening.  The output from the first `mm_pbsa` unit test is 
```
$ /opt/amber/bin/mm_pbsa.pl mm_pbsa.in

=>> Init data
    Presuming executables of amber suite to be in /opt/amber/bin

=>> Reading input parameters
    Found VERBOSE => 0
    Found PARALLEL => 0
    Found PREFIX => test
    Found PATH => ./
    Found START => 1
    Found STOP => 4
    Found OFFSET => 1
    Found COMPLEX => 1
    Found RECEPTOR => 1
    Found LIGAND => 1
    Found COMPT => ../test_com_mbondi2.prmtop
    Found RECPT => ../test_rec_mbondi2.prmtop
    Found LIGPT => ../test_lig_mbondi2.prmtop
    Found GC => 1
    Found AS => 0
    Found DC => 0
    Found MM => 0
    Found GB => 0
    Found PB => 0
    Found MS => 0
    Found NM => 0
    Found BOX => YES
    Found NTOTAL => 4211
    Found NSTART => 1
    Found NSTOP => 4
    Found NFREQ => 1
    Found NUMBER_REC_GROUPS => 1
    Found RSTART => 1
    Found RSTOP => 2199
    Found NUMBER_LIG_GROUPS => 1
    Found LSTART => 2200
    Found LSTOP => 4211
    Found NUMBER_MUTANT_GROUPS => 3
    Found MUTANT_ATOM1 => 1480
    Found MUTANT_ATOM2 => 0
    Found MUTANT_KEEP => 1486
    Found MUTANT_REFERENCE => 1477
    Found MUTANT_ATOM2 => 1498
    Found MUTANT_ATOM1 => 1494
    Found MUTANT_KEEP => 1500
    Found MUTANT_REFERENCE => 1492
    Found MUTANT_ATOM1 => 1552
    Found MUTANT_ATOM2 => 0
    Found MUTANT_KEEP => 1562
    Found MUTANT_REFERENCE => 1549
    Found TRAJECTORY => ./1Z92_vac.mdcrd

=>> Checking sanity
    Checking GENERAL
    Checking GC
    Checking TRAJ

=>> Creating input
    make_crd input

=>> Creating coordinates
    Executing makecrd

 AmberTools16 Module: make_crg_hg

 usage: make_crd_hg <  trajectory_file

 Box info found:    92.153000000000006        92.153000000000006        92.153000000000006
 Box info found:    92.153000000000006        92.153000000000006        92.153000000000006
 Box info found:    92.153000000000006        92.153000000000006        92.153000000000006
 Box info found:    92.153000000000006        92.153000000000006        92.153000000000006
 Box info found:    92.153000000000006        92.153000000000006        92.153000000000006
Note: The following floating-point exceptions are signalling: IEEE_UNDERFLOW_FLAG IEEE_DENORMAL
```

**Comment by hainm** [comment]
you can compare the test before and after your change to amber master branch. If both give the same error, you can safety ignore it (it's author's responsibility in this case).

**Comment by multiplemonomials** [comment]
On further review, I don't actually know if the floating point exceptions are what are causing the problem.  `make_crd_hg` exits normally, and never breaks when I run it in GDB.

**Comment by multiplemonomials** [comment]
Yup it's also broken with the configure script-built Amber.  I guess the mm_pbsa author has some bugfixing to do.

**Comment by hainm** [comment]
No, I meant you should compare current master branch and the git commit before you merge your cmake branch.

**Comment by hainm** [comment]
The test for mmpbsa is ok 14 days ago: https://travis-ci.org/Amber-MD/ambertools-ci/builds/256741827
Fail in last 10 days: https://travis-ci.org/Amber-MD/ambertools-ci/builds/258392023

(check the TEST_TASK=mmpbsa).

**Comment by multiplemonomials** [comment]
Hmm it still fails even when I roll back, at least on my WSL test environment.  Maybe I need to try on a different system.

**Comment by hainm** [comment]
which commit? I can try.

**Comment by multiplemonomials** [comment]
OK I figured it out.  It turns out that `mm_pbsa` itself is fine.  The problem was that in the test script, `MMPBSAEXAMPLES` needed to be an absolute path because of how the tests worked, but I had it being set to `dirname $0`, which was a relative path.  

Fixed by master commit 7318426211
